Source/sources/Castro_sources.cpp

```diff
@@ -46,6 +46,13 @@ Castro::source_flag(int src)
 	else
 	    return false;
 
+    case thermo_src:
+        if (time_integration_method == CornerTransportUpwind ||
```

Comment:
Minor thing, but you could reverse this logic to return true for MOL and SDC and false otherwise? It's better future proofed if we add more integration strategies later.

Reply:
done